custom_components/tuya_local/devices/fujicool_yuzu_heatpump.yaml

```diff
@@ -0,0 +1,270 @@
+name: Fujicool-Yuzu Heatpump (F)
```

Comment:
This should be a generic name (such as Heatpump). The branding belongs in a name under products section, coupled with a product id (available the same place as local key, or when performing a local scan eg with tinytuya scan).